recipes/reno/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "reno" %}
+{% set version = "4.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/reno-{{ version }}.tar.gz
+  sha256: 0ebcad207133f4dbb7bc176ec3513fcd8e107a5d9e2f04b5621eb61da1147089
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

This should fix the Windows build.

```suggestion
  number: 0
  entry_points:
    - reno = reno.main:main
```